llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9425,6 +9362,44 @@ void VPWidenLoadRecipe::execute(VPTransformState &State) {
   }
 }
 
+void VPWidenVPLoadRecipe::execute(VPTransformState &State) {
```

Comment:
A general naming comment: `VPWidenVP...` sounds a bit confusing. The original non-EVL/non-VP masked vector loads and stores are also already "vector predicated", and the VP prefix is already prevalently used to denote VPlan. The distinction lies with the additional EVL operand, currently employed to relieve the mask of conveying the tail. How about `VPWidenEVLLoadRecipe` and `VPWidenEVLStoreRecipe`?